Services/CompanyService/Commands/DisablePositionByIdCommand.cs

```diff
@@ -0,0 +1,22 @@
+﻿using LT.DigitalOffice.CompanyService.Commands.Interfaces;
+using LT.DigitalOffice.CompanyService.Repositories.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using System;
+
+namespace LT.DigitalOffice.CompanyService.Commands
+{
+    public class DisablePositionByIdCommand : IDisablePositionByIdCommand
+    {
+        private readonly IPositionRepository repository;
+
+        public DisablePositionByIdCommand([FromServices] IPositionRepository repository)
+        {
+            this.repository = repository;
+        }
+
+        public void Execute(Guid positionId)
+        {
+            repository.DisablePositionById(positionId);
+        }
```

Comment:
what if positionId will be null or empty?

Reply:
There'll be exception in Repository. I've added tests for it.

Reply:
Maybe we should add checks? Like does the user have rights or is he an admin. Use AccessValidator from develop, but don't forget to register it and modify the appsettings.json properly.